File: inputs.js
//Test Variables
//let age = 15;
//let gender = "female";
//let weight = 125;
//let height = 62;

//getting inputs from html elements
const age = document.getElementById("age")
let gender = document.getElementById("gender")
const weight = document.getElementById("weight")
const height = document.getElementById("height")


// Amount of exercise affects the number of calories
let FreqExercise = 1;
let KiloCalories;

if (FreqExercise === 1) {
  KiloCalories = 1600; // 1600-1700
} else if (FreqExercise === 2) {
  KiloCalories = 1800; // 1800-1900
} else {
  KiloCalories = 2000; // 2000-2100
}

// FoodToday object to store food data
let FoodToday = {
  Name: [],
  Category: [],
  Calories: []
};

// Food class
class Food {
  constructor(name, category, calories) {
    this.name = name;
    this.category = category;
    this.calories = calories;

    // Adding to the FoodToday dictionary
    FoodToday.Name.push(this.name);
    FoodToday.Category.push(this.category);
    FoodToday.Calories.push(this.calories);
  }
}

// Creating a new food object
let potato = new Food("Potato", "Carbs", 600);

// Summing up the calories
const sumCalories = FoodToday.Calories.reduce((acc, curr) => acc + curr, 0);
console.log("Total Calories: " + sumCalories);

// Displaying the data as a table
console.table(FoodToday);

File: inputs.py
username = input(str("What's your name? "))

#getting the demographics of the user
age = int(input(str("Before, we get started, how old are you? ")))
gender = input(str("Are you a female or male? "))
weight = int(input(
    str("We'll get started in a moment! What's your weight in pounds? ")))
height = int(input(str("How tall are you in inches? ")))
FreqExercise = input(
    str("How often do you exercise? Please enter a number. 1:never, 2:moderate, 3:A lot"))

#FreqExercise is necessary in order to get the next value, in case if people don't get it
while FreqExercise != "1" and FreqExercise != "2" and FreqExercise != "3":
  FreqExercise = input(
      str("Please enter a NUMBER from 1-3, where 1:never, 2:moderate, 3:A lot indiciating your exercise level. "))

#changes inputs to integers
FreqExercise = int(FreqExercise)

#testing testing
age = 15
gender = "female"
weight = 125
height = 62

#amount of exercise affects the number of calories
if FreqExercise == 1:
  KiloCalories = 1600#1600-1700
elif FreqExercise == 2:
  KiloCalories = 1800#1800-1900
else:
  KiloCalories = 2000#2000-2100


#To make storing more convenient for food objects
class Food:
  def __init__(self, name, category, calories):
    self.name = name
    self.category = category
    self.calories = calories

    #adding to dictionary in order to create a table later on / just having the whole data set
    FoodToday['Name'].append(self.name)
    FoodToday['Category'].append(self.category)
    FoodToday['Calories'].append(self.calories)

potato = Food("Potato", "Carbs", 600)
print(FoodToday)
df = pd.DataFrame.from_dict(FoodToday)
print(df)
